tendrl/ceph_integration/objects/ecprofile/__init__.py

```diff
@@ -0,0 +1,30 @@
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.ceph_integration import objects
+
+
+class ECProfile(objects.CephIntegrationBaseObject):
+    def __init__(self, name=None, k=None,
+                 m=None, plugin=None,
+                 directory=None,
+                 ruleset_failure_domain=None,
+                 *args, **kwargs):
+        super(ECProfile, self).__init__(*args, **kwargs)
+
+        self.value = 'clusters/%s/ec_profiles/%s'
```

Comment:
how does this object name in value "ec_profiles" match the one you have defined in yaml? It should be "ECProfiles" (which is plural of what is defined in yaml).

Please rectify other object names too.